pyvcloud/vcd/org.py

```diff
@@ -638,3 +642,134 @@ def update_org(self, org_name, is_enabled=None):
                                                 EntityType.ADMIN_ORG.value)
         return org_admin_resource
+
+    def create_org_vdc(self,
+                       vdc_name,
+                       provider_vdc_name,
+                       description='',
+                       allocation_model='AllocationVApp',
+                       cpu_units='MHz',
+                       cpu_allocated=0,
+                       cpu_limit=0,
+                       mem_units='MB',
+                       mem_allocated=0,
+                       mem_limit=0,
+                       nic_quota=0,
+                       network_quota=0,
+                       vm_quota=0,
+                       storage_profiles=[],
+                       resource_guaranteed_memory=None,
+                       resource_guaranteed_cpu=None,
+                       vcpu_in_mhz=None,
+                       is_thin_provision=None,
+                       network_pool_name=None,
+                       uses_fast_provisioning=None,
+                       over_commit_allowed=None,
+                       vm_discovery_enabled=None,
+                       is_enabled=True):
+        """
+        Create Organization VDC in the current Org.
+        :param vdc_name (str): The name of the new org vdc.
+        :param provider_vdc_name (str): The name of the new provider vdc.
+        :param description (str): The description of the new org vdc.
+        :param allocation_model (str): The allocation model used by this vDC. One of AllocationVApp, AllocationPool or ReservationPool.
+        :param cpu_units (str): The cpu units compute capacity allocated to this vDC. One of MHz or GHz
+        :param cpu_allocated (int): Capacity that is committed to be available.
+        :param cpu_limit (int): Capacity limit relative to the value specified for Allocation.
+        :param mem_units (str): The memory units compute capacity allocated to this vDC. One of MB or GB.
+        :param mem_allocated (int): Memory capacity that is committed to be available.
+        :param mem_limit (int): Memory capacity limit relative to the value specified for Allocation.
+        :param nic_quota (int): Maximum number of virtual NICs allowed in this vDC. Defaults to 0, which specifies an unlimited number.
+        :param network_quota (int): Maximum number of network objects that can be deployed in this vDC. Defaults to 0, which means no networks can be deployed.
+        :param vm_quota (int): The maximum number of VMs that can be created in this vDC. Defaults to 0, which specifies an unlimited number.
+        :param storage_profiles: List of provider vDC storage profiles to add to this vDC.
+            Each item is a dictionary that should include the following elements:
+                name: (string) name of the PVDC storage profile.
+                enabled: (bool) True if the storage profile is enabled for this vDC.
+                units: (string) Units used to define limit. One of MB or GB.
```

Comment:
'Units used to define the limit, in MB or GB' 
might be better.

Reply:
most of this documentation is copied from the vCD API schema for accuracy